contribs/gmf/examples/locationChooser.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html ng-app='app'>
+  <head>
+    <title>Simple GeoMapFish example</title>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="initial-scale=1.0, user-scalable=no, width=device-width">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <link rel="stylesheet" href="../../../node_modules/openlayers/css/ol.css" type="text/css">
+    <style>
+      gmf-map > div {
+        width: 600px;
+        height: 400px;
+      }
+    </style>
+  </head>
+  <body ng-controller="MainController as ctrl">
+    <div>
+        <span>Move to:</span>
+        <gmf-location-chooser
```

Comment:
The convention to use `gmf-componentname` rather than `gmf-component-name`. In this way we never have more than three words in tag and attribute names. So in this case we will have:


```html
  <gmf-locationchooser
      gmf-locationchooser-map="ctrl.map" gmf-locationchooser-locations="ctrl.locations">
  </gmf-locationchooser>
```